eng/Versions.props

```diff
@@ -223,7 +223,7 @@
     <MicrosoftOwinSecurityCookiesVersion>3.0.1</MicrosoftOwinSecurityCookiesVersion>
     <MicrosoftOwinTestingVersion>3.0.1</MicrosoftOwinTestingVersion>
     <MicrosoftWebAdministrationVersion>11.1.0</MicrosoftWebAdministrationVersion>
-    <MicrosoftWebXdtVersion>1.4.0</MicrosoftWebXdtVersion>
+    <MicrosoftWebXdtVersion>7.0.0-preview.22419.1</MicrosoftWebXdtVersion>
```

Comment:
If we actually need to change Version.Details.props, this should be moved to the `Label="Automated"` property group. But I'd rather we just change this one line and leave the other file alone if possible. I don't see a solid reason to add a new `darc` subscription but may be missing something.

Reply:
OK, we could remove the explicit version here, if not required anymore.

Reply:
We need the version because we use the package. It's just the _location_ w/in this file I'm objecting to now that you've convinced me Version.Details.props must change.

Reply:
Sounds good - thanks!